literature.bib

```diff
@@ -4480,6 +4600,77 @@ @article{TKB+:SCP14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2013 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{SLSA:SIGPLAN13,
+	author = {Schulze, Sandro and Liebig, J\"{o}rg and Siegmund, Janet and Apel, Sven},
+	title = {{Does the Discipline of Preprocessor Annotations Matter?: A Controlled Experiment}},
+	journal = SIGPLAN,
+	volume = {49},
+	number = {3},
+	year = 2013,
+	issn = {0362-1340},
+	pages = {65--74},
+	numpages = {10},
+	url = {http://doi.acm.org/10.1145/2637365.2517215},
+	doi = {10.1145/2637365.2517215},
+	acmid = {2517215},
+	publisher = ACM,
+	address = NY,
+	keywords = {c preprocessor, controlled experiment, disciplined annotations, program comprehension, variability},
+}
+
+@inproceedings{KKT:ACM13,
+	author = {Timo {Kehrer} and Udo {Kelter} and Gabriele {Taentzer}},
+	booktitle = ASE,
+	publisher = ACM,
+	title = {{Consistency-Preserving Edit Scripts in Model Versioning}},
+	year = 2013,
+	pages = {191--201},
+	pb-tags = {semantic lifting},
+	keywords = {configuration management, software tools, consistency preserving edit scripts, model versioning, software development, internal model representations, automatic procedure, model merging, model comparison, model patching, software tools, Unified modeling language, Merging;Semantics, Concrete, Adaptation models, Abstracts, Syntactics},
+	doi = {10.1109/ASE.2013.6693079}
+}
+
+% For this entry, I could not find any address.
```

Comment:
publisher and address is not needed for journals anyway

so in future, not need to document for those

Reply:
"I" is unclear

Reply:
Ok didn't know that. That makes things a lot easier! 😃 I removed the comment.